examples/esp-todomvc-react/js/footer.jsx

```diff
@@ -0,0 +1,64 @@
+/*global React*/
+var app = app || {};
+
+(function () {
+    "use strict";
+
+    app.Footer = React.createClass({
+
+        handleClear: function () {
+            this.props.router.publishEvent("todoList", "clearCompleted", {});
+        },
+
+        render: function () {
+            var footer = this.props.model;
+            if (!footer.visible) {
+                return null;
+            }
+            var incompleteCountLabel = app.Utils.pluralize(footer.incompleteCount, "item");
+            var clearButton = null;
+            if (footer.clearButtonVisible) {
+                clearButton = (
+                    <button
+                        className="clear-completed"
+                        onClick={this.handleClear}>
+                        Clear completed
+                    </button>
+                );
+            }
+            return (
+                <footer className="footer">
+					<span className="todo-count">
+						<strong>{footer.incompleteCount}</strong> {incompleteCountLabel} left
+					</span>
+                    <ul className="filters">
+                        <li>
+                            <a
+                                href="#/"
+                                className={React.addons.classSet({selected: footer.filter === app.model.Filter.all})}>
+                                All
+                            </a>
+                        </li>
+                        {" "}
```

Comment:
This was copied from the React TodoMVC examples. Can change if there's a better way